### PR TITLE
Catholics

### PR DESCRIPTION
**Pull request by Gr770**

### CODE REVIEW

File: common/council_positions/00_council_positions.txt [hunk 1]
```diff
@@ -208,6 +208,342 @@ councillor_court_chaplain = {
...
+					scope:councillor_liege.faith = faith:catholic
+				}
+				desc = {
+					first_valid = {
+						triggered_desc = {
+							trigger = {
+								scope:councillor_liege.culture_group = culture_group:goidelic_group
+								scope:councillor_liege.highest_held_title_tier = tier_county
+							}
+							desc = count_male_theo_cath_goidelic
```
Comment (Viridianus) [inline common/council_positions/00_council_positions.txt:222]: Naming conventions here are off? Do (like here) or don't (like 7 lines later) we mark "theo"?

**Review by Viridianus — COMMENTED** [review]
Why are we doing it like this even? Shouldn't we, like, have a simple reference in 00_council_positions.txt and then customloc it?

**Comment by Gr770** [comment]
councilor_liege seems to be council file only. It's not listed as a general ck3 scope.

**Comment by Viridianus** [comment]
Yeah, cause otherwise liege suffices
--
Отправлено из Mail.ru для Android воскресенье, 07 февраля 2021г., 22:14 +03:00 от Gr770  notifications@github.com :

**Pull request merged**